glib/src/subclass/signal.rs

```diff
@@ -6,17 +6,18 @@ use crate::SignalFlags;
 use crate::Type;
 use crate::Value;
 
-use std::fmt;
 use std::ptr;
 use std::sync::Mutex;
+use std::{fmt, num::NonZeroU32};
 
 /// Builder for signals.
 #[allow(clippy::type_complexity)]
 pub struct SignalBuilder<'a> {
     name: &'a str,
     flags: SignalFlags,
-    arg_types: &'a [Type],
-    ret_type: Type,
+    param_types: &'a [Type],
+    return_type: Type,
+    static_scope: bool,
```

Comment:
Each parameter and the return type can be static scope individually

Reply:
Right, which means that the static_scope should be per param_type & return_type as well I guess?

Reply:
I meant for the SignalQuery part

Reply:
Yes